flatgfa/src/packedseq.rs

```diff
@@ -204,20 +285,46 @@ pub fn create_slice(vec: &PackedVec, s: std::ops::Range<usize>) -> PackedSlice<'
 
 /// Returns a vector containing the base pairs referenced by `slice`
 pub fn get_slice_seq(slice: PackedSlice<'_>) -> Vec<Nucleotide> {
-    slice.vec_ref.get_range(slice.span)
+    slice.vec_ref.as_ref().get_range(slice.span)
+}
+
+pub fn total_bytes(seq: &PackedSeqView) -> usize {
```

Comment:
This seems like it should be a method on `PackedSeqView`. It would also be great to clarify (in docs, and/or the function name) that this is about the _file_ size, i.e., the number of bytes you need to hold the result of `dump`.